### ISSUE TITLE
Acceptance test is failing on Debian

### ISSUE THREAD

**Issue by jordansissel** [elastic/logstash#6198]
https://logstash-ci.elastic.co/job/elastic+logstash+5.0+multijob-acceptance/

This job has been failing since October 26.

**Comment by jordansissel** [comment]
Failures from first failed build:

```
Failures:

  1) artifacts operation behaves like installable is running on ubuntu-1204
     Failure/Error: expect(logstash).to be_running
       expected #<ServiceTester::Artifact:0x5b080f3a @host="127.0.0.1:2200", @client=#<ServiceTester::UbuntuCommands:0x773cbf4f>, @options={"host"=>"ubuntu-1204", "type"=>"debian"}> to be running
     Shared Example Group: "installable" called from /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/lib/artifact_operation_spec.rb:12
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/shared_examples/installed.rb:18:in `(root)'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/Rakefile:66:in `(root)'

  2) artifacts operation behaves like updated can be updated an run on ubuntu-1204
     Failure/Error: expect(logstash).to be_running
       expected #<ServiceTester::Artifact:0x5b080f3a @host="127.0.0.1:2200", @client=#<ServiceTester::UbuntuCommands:0x773cbf4f>, @options={"host"=>"ubuntu-1204", "type"=>"debian"}> to be running
     Shared Example Group: "updated" called from /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/lib/artifact_operation_spec.rb:13
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/shared_examples/updated.rb:25:in `(root)'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/Rakefile:66:in `(root)'

  3) artifacts operation behaves like installable is running on ubuntu-1404
     Failure/Error: expect(logstash).to be_running
       expected #<ServiceTester::Artifact:0x7fe083b1 @host="127.0.0.1:2201", @client=#<ServiceTester::UbuntuCommands:0x23c388c2>, @options={"host"=>"ubuntu-1404", "type"=>"debian"}> to be running
     Shared Example Group: "installable" called from /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/lib/artifact_operation_spec.rb:12
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/shared_examples/installed.rb:18:in `(root)'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/Rakefile:66:in `(root)'

  4) artifacts operation behaves like updated can be updated an run on ubuntu-1404
     Failure/Error: expect(logstash).to be_running
       expected #<ServiceTester::Artifact:0x7fe083b1 @host="127.0.0.1:2201", @client=#<ServiceTester::UbuntuCommands:0x23c388c2>, @options={"host"=>"ubuntu-1404", "type"=>"debian"}> to be running
     Shared Example Group: "updated" called from /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/lib/artifact_operation_spec.rb:13
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/acceptance/spec/shared_examples/updated.rb:25:in `(root)'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-acceptance/label/metal/suite/debian/qa/Rakefile:66:in `(root)'

Finished in 21 minutes 41 seconds (files took 1.21 seconds to load)
69 examples, 4 failures, 6 pending
```

**Comment by ph** [comment]
On thing that changed between releases 5.0.0 and 5.0.1 is the bump to pleaserun 

https://github.com/elastic/logstash/commit/4b3b1ab29ea21a7a7d95637f49fa7e2e0c9efad8#diff-efbf03f6386a8f9610003e5858706663L514

@suyograo and myself have installed the 5.0.1 package on the host and we are able to run it as a CLI but it refuses to start with `service start` in the syslog we see that it keep restarting the logstash program.

**Comment by jordansissel** [comment]
Problem is a bug on the upstart job generated by pleaserun. I'm on it.

**Comment by jordansissel** [comment]
pleaserun 0.0.27 published and I added test coverage for non-root users for jobs like the ones Logstash creates.

**Comment by suyograo** [comment]
Fixed in #6199
